tox.ini

```diff
@@ -23,6 +23,10 @@ deps =
 commands =
      tests-!windows-!mac: sh -c "pybamm_install_jax"  # install jax, jaxlib for ubuntu
      unit-!windows-!mac: sh -c "pybamm_install_jax"  # install jax, jaxlib for ubuntu
+     odes-!windows-!mac: sh -c "rm -f /home/*/.local/lib/*sundials*"
+     odes-!windows-!mac: sh -c "rm -f -r /home/*/.local/include/*sundials*"
+     odes-!windows-!mac: sh -c "pip uninstall --yes scikits.odes"
+     odes-!windows-!mac: sh -c "pip install wget"
```

Comment:
Is it possible to club these commands in a single line or as a list? If not, then this should be good too.

Reply:
I've been able to do that, but now while i'm testing it using `tox -e odes` i'm getting the same error logs I was getting before. I would also verify this by running scheduled tests.

Reply:
@Saransh-cpp scheduled tests are having same [logs](https://github.com/arjxn-py/PyBaMM/actions/runs/5121164913), so it is not likely an issue with my local system.